### PR TITLE
Db framework

### PR DESCRIPTION
**Pull request by AndriyPovzun**

### CODE REVIEW

File: SQLAlchemy_db_connector/repositories/orders_repository.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+from SQLAlchemy_db_connector import connection_db
+from SQLAlchemy_db_connector.models.orders import Orders
+from SQLAlchemy_db_connector.models.products import Products
+from sqlalchemy import select, delete, update
+
+
+class OrdersRepository:
```
Comment (bohdandovbsyh) [inline SQLAlchemy_db_connector/repositories/orders_repository.py:7]: does it work in separate class with pony?

File: SQLAlchemy_db_connector/repositories/orders_repository.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    def __init__(self):
+        self.__session = connection_db.session
+        self.__result_list = []
+
+    def get_all_orders(self) -> list:
+        all_orders = self.__session.query(Orders).all()
+        return all_orders
+
+    def get_by_id(self, id_value: int):
```
Comment (bohdandovbsyh) [inline SQLAlchemy_db_connector/repositories/orders_repository.py:16]: if I remember correctly all pony methods required decorator

Comment (AndriyPovzun) [reply SQLAlchemy_db_connector/repositories/orders_repository.py:16]: I don't know, all work correctly. No errors, warnings, or hints when program run

File: SQLAlchemy_db_connector/repositories/orders_repository.py (hunk 1 above)
Comment (AndriyPovzun) [reply SQLAlchemy_db_connector/repositories/orders_repository.py:7]: what is a pony? I built the framework according to the lesson and spent a lot of time studying the documentation

Comment (AndriyPovzun) [reply SQLAlchemy_db_connector/repositories/orders_repository.py:7]: I understand, pony in the next lesson. I will watch and implement in this framework

File: SQLAlchemy_db_connector/repositories/orders_repository.py (hunk 2 above)
Comment (bohdandovbsyh) [reply SQLAlchemy_db_connector/repositories/orders_repository.py:16]: Sorry, my fault. I thought it is PonyOrm implementation. Comments are not relevant

**Pull request merged**